### PR TITLE
fixed review indicator and export button alignments

### LINKED ISSUE #1251: Tables data CSV export button not aligned when data entry is unlocked
Screenshot 2022-04-12 at 16 59 55

### PR DESCRIPTION
**Pull request by rfontanarosa**
close #1251
Schermata 2022-04-27 alle 13 57 13

### CODE REVIEW

File: client/components/ButtonTableExport/ButtonTableExport.scss [hunk 1]
```diff
@@ -6,6 +6,10 @@
   top: -24px;
   font-size: $font-xs;
 
+  &.in-review {
+    margin-right: 32px;
```
Comment (minotogna) [inline client/components/ButtonTableExport/ButtonTableExport.scss:10]: use spacing-m

File: client/components/ButtonTableExport/ButtonTableExport.tsx [hunk 2]
```diff
@@ -22,7 +24,9 @@ const ButtonTableExport: React.FC<Props> = (props) => {
 
   return (
     <CSVLink
-      className={`fra-table__btn-export btn-xs btn-primary${isLocked && !disabled ? '' : ' disabled'}`}
+      className={`fra-table__btn-export ${inReview ? 'in-review' : ''} btn-xs btn-primary${
+        isLocked && !disabled ? '' : ' disabled'
+      }`}
```
Comment (minotogna) [inline client/components/ButtonTableExport/ButtonTableExport.tsx:29]: use classNames

File: client/styles/fraTable.scss [hunk 3]
```diff
@@ -142,6 +142,10 @@
   right: 0;
 }
 
+.fra-table__review-cell {
+  width: 32px;
```
Comment (minotogna) [inline client/styles/fraTable.scss:146]: use spacing-m

**Review by minotogna — CHANGES_REQUESTED** [review]

File: client/styles/fraTable.scss (hunk 3 above)
Comment (rfontanarosa) [reply client/styles/fraTable.scss:146]: the problem is that fra-table__review-cell class is under styles/fraTable.css and spacing-m is under style/_partials/spacing.

another strange thing is that config.scss is replicated, like buttons.scss and status.scss...

Schermata 2022-04-28 alle 09 18 25

**Review by minotogna — APPROVED** [review]

**Pull request merged**